crates/lib/src/qpu/client.rs

```diff
@@ -46,6 +49,13 @@ impl Qcs {
     pub async fn load() -> Result<Self, LoadError> {
         ClientConfiguration::load_default()
             .await
+            .or_else(|_| {
+                #[cfg(feature = "tracing")]
+                tracing::info!(
+                    "No QCS client configuration found, only generic QVMs will be accessible."
+                );
+                Ok(ClientConfiguration::default())
+            })
```

Comment:
I initially added this to `get_qc` in pyQuil specifically, but it's probably better to do it here, since this library can be used without a full blown QCS config as well.

Reply:
A separate issue is how this client is technically buried in the `qpu` module, despite being used by the Python bindings for everything. As a follow-up, it probably makes sense to lift this towards the root module and use it as the client for everything in the Rust crate as well.

Reply:
Agreed

Reply:
Resolving this with a [new issue](https://github.com/rigetti/qcs-sdk-rust/issues/298)